src/superio/nuvoton/nct5104d/nct5104d.h

```diff
@@ -23,13 +23,24 @@
 /* SIO global configuration */
 #define IRQ_TYPE_SEL_CR10   0x10     /* UARTA,UARTB */
 #define IRQ_TYPE_SEL_CR11   0x11     /* SMI,UARTC,UARTD,WDTO */
+#define GLOBAL_OPTION_CR7   0x07
```

Comment:
@piotr-kleins please revert to LDN_SELECT

Reply:
Done